### DISCUSSION TITLE
Localhost is working perfectly but Its not working on live serve.

### DISCUSSION THREAD

**Discussion by satzofficial** [opcodesio/log-viewer#337] [Q&A]
![image](https://github.com/opcodesio/log-viewer/assets/140888681/947e39d3-6d9c-4402-8486-996e4a1f783f)

Congratulation for the ideas, In live server i got the above 404 page, to look the image, I don't know what kind of mistake took on my projects, Kindly suggest the log view for my domain.

**Comment by arukompas** [comment]
hey @satzofficial , please share 
- your `config/log-viewer.php`, 
- the URL you're trying to visit,
- how is production set up? Using Apache/nginx/caddy?

Thanks 🙏

**Reply by satzofficial** [reply]
Thanks for the suggestions @arukompas I have resolved the issue

**Reply by vailsonmarcelo** [reply]
hi. how you solve this problem? I have same issue

**Reply by arukompas** [reply]
Hey @vailsonmarcelo please see this:

https://log-viewer.opcodes.io/docs/3.x/configuration/route-and-domain#production-domains

**Reply by kwadejeffrey** [reply]
Hi, I'm facing same issue but I'm getting 403 forbidden error. 
How do I resolve this? I have set my APP_URL and all

**Reply by arukompas** [reply]
@kwadejeffrey does the `APP_URL` also contain the correct scheme and subdomain? e.g. There's a difference between https:// and http://, and also between http://example.com and http://www.example.com, so make sure to visit the **exact** domain (with the correct scheme) as you've configured it in APP_URL and your apache/nginx configuration.
